docs/opentelemetry/otel-mapping-summary.asciidoc

```diff
@@ -0,0 +1,1164 @@
+
+
+| Peer
+^| ·
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/peer[1]
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| 
+
+
+| Process
+^| <<ecs-process,37>>
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/process[33]
+^| 15
+^| 2
+^| ·
+^| ·
+^| 1
+^| ·
+^| ·
+
+
+| profile
+^| ·
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/profile[1]
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| 
+
+
+| Registry
+^| <<ecs-registry,7>>
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+
+
+| Related
+^| <<ecs-related,4>>
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| 4
+
+
+| Risk information
+^| <<ecs-risk,6>>
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+
+
+| Remote Procedure Call (RPC)
+^| ·
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/rpc[17]
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| 
+
+
+| Rule
+^| <<ecs-rule,10>>
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+
+
+| Server
+^| <<ecs-server,12>>
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/server[2]
+^| 2
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+
+
+| Service
+^| <<ecs-service,11>>
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/service[4]
+^| 2
+^| 2
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+
+
+| Session
+^| ·
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/session[2]
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| 
+
+
+| SignalR
+^| ·
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/signalr[2]
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| 
+
+
+| Source
+^| <<ecs-source,12>>
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/source[2]
+^| 2
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+
+
+| General System
+^| ·
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/system[12]
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| 
+
+
+| Telemetry
+^| ·
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/telemetry[5]
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| 
+
+
+| test
+^| ·
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/test[4]
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| 
+
+
+| Thread
+^| ·
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/thread[2]
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| 
+
+
+| Threat
+^| <<ecs-threat,65>>
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+
+
+| TLS
+^| <<ecs-tls,29>>
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/tls[28]
+^| 26
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+
+
+| Tracing
+^| <<ecs-tracing,3>>
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| 2
+^| 1
+
+
+| URL
+^| <<ecs-url,14>>
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/url[13]
+^| 12
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+
+
+| User
+^| <<ecs-user,7>>
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/user[6]
+^| 6
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+^| ·
+
+
+| User agent
+^| <<ecs-user_agent,4>>
+^| https://opentelemetry.io/docs/specs/semconv/attributes-registry/user_agent[4]
```

Comment:
This is a 404. It should be https://opentelemetry.io/docs/specs/semconv/attributes-registry/user-agent